src/format/Format.cc

```diff
@@ -1390,7 +1390,7 @@ Format::Format::assemble(MemBuf &mb, const AccessLogEntry::Pointer &al, int logS
             out = sb.c_str();
         } else if (doMsec) {
             if (fmt->widthMax < 0) {
-                sb.appendf("%0*ld", fmt->widthMin, tvToMsec(outtv));
+                sb.appendf("%0*ld", fmt->zero && fmt->widthMin >= 0 ? fmt->widthMin : 0, tvToMsec(outtv));
```

Comment:
We should honor `fmt->widthMin` (and pad with spaces) when `fmt->zero` is false, right? I do not think the above code does that.

Reply:
I thought that too. But the above change makes this format effectively identical to the others in surrounding lines, and log tests confirm the output is correct for all zero vs non-zero and min-only vs min+max combinations.

Reply:
logformat:
  ms_8tr=%8tr , ms=08tr=%08tr , ms_8.1tr=%8.1tr , ms_08.1tr=%08.1tr %>rp

> ms_8tr=     339 , ms=08tr=00000339 , ms_8.1tr=   339.0 , ms_08.1tr=000339.0 /patched
> ms_8tr=00000333 , ms=08tr=00000333 , ms_8.1tr=   333.0 , ms_08.1tr=000333.0 /broken

(github mangles the spacing, so it may now show up here but things do line up correctly).